examples/react-quickstart/README.md

```diff
@@ -14,17 +14,17 @@ To learn more about XMTP and get answers to frequently asked questions, see [FAQ
 
 ### Limitations
 
-This example app demonstrates features of the React SDK, but it isn't a complete solution. For example, the list of conversations doesn't update when new messages arrive in existing conversations.
+This example app demonstrates features of the React XMTP client SDK, but it isn't a complete solution. For example, the list of conversations doesn't update when new messages arrive in existing conversations.
 
 ## Developing
 
-After installing the dependencies, run `yarn dev` to start developing.
+Install the [React client SDK and requirements](/packages/react-sdk/README.md) and then run `yarn dev` to start developing.
 
 ## Useful commands
 
 - `yarn build`: Builds the example app
 - `yarn clean`: Removes `node_modules`, `dist`, and `.turbo` folders
-- `yarn dev`: Launches the example app in development mode
+- `yarn dev`: Launches the example app in development mode at `http://127.0.0.1:5173`
```

Comment:
i don't think it's necessary to add the IP address and port here. it's in the terminal when running `yarn dev` and the port can be changed.